content/about/resources.en.md

```diff
@@ -21,20 +21,22 @@ Guidelines for the visual display of the Node.js mark are described in the [Visu
 <!--lint enable nodejs-links-->
 
 <table class="resources-logos">
-  <tr>
-    <td class="bg-white"><a href="/static/images/logos/nodejs-new-pantone-black.ai"><img src="/static/images/logos/nodejs-new-pantone-black.svg" alt="Node.js on light background"/></a></td>
-    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-pantone-white.ai"><img src="/static/images/logos/nodejs-new-pantone-white.svg" alt="Node.js on dark background"/></a></td>
-  </tr>
-  <tr>
-    <td><a href="/static/images/logos/nodejs-new-pantone-black.ai">Node.js standard AI</a></td>
-    <td><a href="/static/images/logos/nodejs-new-pantone-white.ai">Node.js reversed AI</a></td>
-  </tr>
-  <tr>
-    <td class="bg-white"><a href="/static/images/logos/nodejs-new-black.ai"><img src="/static/images/logos/nodejs-new-black.svg" alt="Node.js on light background"/></a></td>
-    <td class="bg-node-gray"><a href="/static/images/logos/nodejs-new-white.ai"><img src="/static/images/logos/nodejs-new-white.svg" alt="Node.js on dark background"/></a></td>
-  </tr>
-  <tr>
-    <td><a href="/static/images/logos/nodejs-new-black.ai">Node.js standard with less color AI</a></td>
-    <td><a href="/static/images/logos/nodejs-new-white.ai">Node.js reversed with less color AI</a></td>
-  </tr>
+  <tbody>
```

Comment:
This seems like a semantic change. Should be keep it in this PR?

Reply:
True, gonna drop this change.